### DISCUSSION TITLE
UI / Display using Holoviz Panel

### DISCUSSION THREAD

**Discussion by hagrid67** [flatland-association/flatland-rl#4] [Ideas]
Panel is a bit more like a webpage than a Jupyter Notebook.  It shares a lot of modules with Jupyter.  I suggest we try to move some of the rendering code into Panel.  (I often find Notebooks become messy too quickly; I have been using Panel for some personal projects)

Panel allows you to make something a bit more polished and reusable than a notebook; more like a Dashboard.  It takes a bit more coding than a notebook to get something working.

**Comment by aiAdrian** [comment]
[Holoviz Panel ](https://panel.holoviz.org/gallery/index.html)?
